docs-kits/kits/Manufacturing as a Service Kit/page_adoption-view.md

```diff
@@ -11,140 +11,94 @@ sidebar_position: 2
+![Link to the journal paper: <https://www.researchgate.net/publication/ 373383262_Building_a_Digital_Manufacturing_as_a_Service_Ecosystem_for_Catena-X>](resources/Architekturteaser.png)
 
-![Link to the journal paper: <https://www.researchgate.net/publication/ 373383262_Building_a_Digital_Manufacturing_as_a_Service_Ecosystem_for_Catena-X>](resources/Architekturteaser.png)  
-*Building a Digital Manufacturing as a Service Ecosystem for Catena-X[^1]*
+*Building a Digital Manufacturing as a Service Ecosystem for Catena-X[^1].*
 
 ## Semantic Models
 
-The knowledge of manufacturing capabilities is based on extensive knowledge from different distributed sources in
-various fields of expertise and interoperability of systems utilizing and managing this information is a major issue.
-The reason for this is that the majority of these systems use their own proprietary data structures and vaguely describe
-semantics, as well as different interpretations of the terms used and applied to describe the capabilities. For this
-reason, the CDME architecture does not rely on standardized semantics. However, basic concepts are defined as aspect
-models, to provide a common semantics where appropriate.
+Knowledge of manufacturing capabilities is based on extensive knowledge from different distributed sources, in different disciplines, and the interoperability of systems that use and manage this information is a major challenge. The reason for this is that the majority of these systems use their own proprietary data structures and vaguely describe semantics, as well as different interpretations of the terms used and applied to describe the capabilities. For this reason, the CDME architecture does not rely on standardized semantics. However, basic concepts are defined as aspect models, to provide a common semantics where appropriate.
 
 ### Manufacturing Capability Model
 
-An aspect model representing manufacturing capabilities, based on the concepts for products, processes, resources and
-capabilities, as well as their relations to each other. Sharing information about the required and available
-manufacturing capabilities in an interoperable manner is fundamental for any MaaS solution providing basic and
-value-added services such as automated search and matchmaking.
-Link to the semantic data
-model: <https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.manufacturing_capability/2.0.0>
+An aspect model representing manufacturing capabilities, based on the concepts for products, processes, resources and capabilities, as well as their relations to each other. Sharing information about the required and available manufacturing capabilities in an interoperable manner is fundamental for any MaaS solution providing basic and value-added services such as automated search and matchmaking.
+
+Link to the semantic data model: <https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.manufacturing_capability/2.0.0>
```

Comment:
This should be aligned with CX-0115 Manufacturing Capability Exchange (v1.0.0), as it refers to  urn:samm:io.catenax.manufacturing_capability:3.1.0 as the standardized data model.

Edit: The data model v3.1.0 seems already to be referenced in the other sections of this KIT, too.

Reply:
model references have been updated